src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(error_generic_member_access)]
+#![feature(error_in_core)]
```

Comment:
For readers: details of the feature can be found here: https://github.com/rust-lang/rust/issues/103765